examples/slu/slurp/scripts/eval_slurp.sh

```diff
@@ -0,0 +1,16 @@
+DATA_DIR="./slurp_data"
```

Comment:
Same as above. Better yet, dont put recipes like this. Put the instructions for eval in the docstring for the eval script.